play-json/shared/src/main/scala/JsResult.scala

```diff
@@ -3,13 +3,14 @@
  */
 package play.api.libs.json
 
-import Json._
-
 /**
  * The result for a successful parsing.
  */
 case class JsSuccess[T](value: T, path: JsPath = JsPath()) extends JsResult[T] {
   def get: T = value
+
+  val isSuccess = true
```

Comment:
"Statically" declare `isSuccess`/`isError` according the sub-types of `JsResult` (see `Try` impl)